lib/module.js

```diff
@@ -12,3 +12,5 @@ export default /* async */ function (moduleOptions) {
 }
 
 const DEFAULTS = {}
+
+export const meta = require('../package.json')
```

Comment:
I think nuxt/esm picks default export so this field won't be picked

Reply:
Good point. Should we adjust docs?

Is this looking for the key to be set on the function prototype? 🤔

Reply:
Yep: https://github.com/nuxt/nuxt.js/blob/dev/packages/core/src/module.js#L194

Docs are also mentioning same: https://nuxtjs.org/guide/modules/#write-a-basic-module

Reply:
I missed the `module.` 🤦